src/cryptoadvance/specter/server_endpoints/controller.py

```diff
@@ -145,6 +148,33 @@ def selfcheck():
         app.login("admin")
 
 
+@app.before_request
+def slow_request_detection():
+    """ """
+    g.start = time()
+
+
+@app.after_request
+def after_request(response):
+    diff = time() - g.start
+    if (
+        (response.response)
+        and (200 <= response.status_code < 300)
+        and (response.content_type.startswith("text/html"))
+    ):
+        treshold = app.config["REQUEST_TIME_WARNING_TRESHOLD"]
+        if diff > treshold:
+            flash(
+                _(
+                    "The request before this one took {} seconds which is longer than the threshold ({}). Checkout the perfomance-improvement-hints in the documentation".format(
```

Comment:
My proposal here: "The last request took very long. Check out the performance improvement-hints in the documentation"
I would also add a link to `performance.md` on GitHub (remark, one needs the Markup class from flask to show a href in flash message: https://stackoverflow.com/questions/21248718/how-to-flashing-a-message-with-link-using-flask-flash)